Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
     platforms: [
         // supported
         // Linux
-        .macOS(.v11),
+        .macOS(.v13),
```

Comment:
Is this required because that'll be "SemVer major", i.e. would require us to tag 0.4.0

Reply:
If this is required, we'd need to raise all platforms accordingly.

Reply:
@FranzBusch what's the latest recommendation w.r.t. platform reqs?

Reply:
1. [swift-otel ](https://github.com/swift-otel/swift-otel/blob/main/Package.swift#L147) is macOS 13 / iOS 16
2. [swift-prometheus](https://github.com/swift-server/swift-prometheus/blob/main/Package.swift#L20) is macOS 13 / iOS 16
3. Rest of the projects don't set a supported platform and rely on a default which is something very old.